managed/services/telemetry/transform.go

```diff
@@ -86,6 +86,27 @@ func transformToJSON(config *Config, metrics []*pmmv1.ServerMetric_Metric) ([]*p
 	}, nil
 }
 
+func transformStripValues(config *Config, metrics []*pmmv1.ServerMetric_Metric) ([]*pmmv1.ServerMetric_Metric, error) {
```

Comment:
I think we need a better naming here, maybe `transformValuesPresented` or something else

Reply:
I renamed it to `transformExportValues` :)